### DISCUSSION TITLE
如何修改首页文章目录封面图的默认高度？

### DISCUSSION THREAD

**Discussion by guowei-gong** [CaiJimmy/hugo-theme-stack#439] [Q&A] [answered]
(*´▽｀)ノノ 你好，我希望文章列表的默认图片高度变得更长，预期如下图，绿色框是默认高度，蓝色框是我期望的高度，红色部分是增加的宽度：
![image](https://user-images.githubusercontent.com/62499904/147021366-c422f576-a6d0-4d3a-973a-5dfa1cfdda81.png)

请问可以做到吗？如果可以我应该怎么做？

**Comment by zhixuan2333** [comment, marked as answer]
https://github.com/CaiJimmy/hugo-theme-stack/blob/359e4b34c1f96beeda472cddb54cea98522b2c14/assets/scss/partials/article.scss#L24-L38

修改`height`。
